packages/sdk-components-react/src/image.ws.tsx

```diff
@@ -37,9 +37,6 @@ export const meta: WsComponentMeta = {
   order: 0,
 };
 
-// "loader" is our internal prop not intended to show up in the props panel
-const { loader, ...publicProps } = props;
```

Comment:
I think we could have simpler solution. Why it is generated at all? It's prop of internal component.
```
type Props = Omit<ComponentPropsWithoutRef<typeof WebstudioImage>, 'loader'>;
```

Reply:
damn, true

Reply:
ah noo, this way you can't use the loader prop at all

Reply:
the problem is that it's a valid prop, one could use it if needed over code

Reply:
we don't have loader prop in props panel

Reply:
actually in this particular case, loader isn't a valid prop, it is an internal one, so your solution is correct here, but generally I think we will have a situation where we need ignoredProps

Reply:
Explicitly listing is hard. I'd just ignore internally on generation.